pkg/operator/operator.go

```diff
@@ -496,12 +505,12 @@ func (o *Operator) sync(key string) error {
 	return nil
 }
 
-func (o *Operator) reportError(err error, taskName string) {
+func (o *Operator) reportError(err error, failedTaskReason string) {
 	klog.Infof("ClusterOperator reconciliation failed (attempt %d), retrying. Err: %v", o.failedReconcileAttempts+1, err)
```

Comment:
Following my comment below, we can avoid logging again `err`.

```suggestion
	klog.Infof("ClusterOperator reconciliation failed (attempt %d), retrying", o.failedReconcileAttempts+1)
```